microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -59,6 +59,10 @@ The following list provides update details:
 * Updates of the RPMs on a non-OSTree system such as {op-system} are also supported.
 * Updates from the 4.14 version are supported.
 
+[id="microshift-4-15-version-logs-at-start"]
+==== Version logged at start up
+Previously, {microshift-short} did not log its version at start up. The absence of version information made debugging certain scenarios difficult because the update path was unknown. Now, the {microshift-short} version is logged at start up and available to commands such as `journalctl -u microshift | grep "Version"`. (link:https://issues.redhat.com/browse/OCPBUGS-19540[*OCPBUGS-19540*])
```

Comment:
```suggestion
Previously, {microshift-short} did not log its version at start up. The absence of version information made debugging certain scenarios difficult because the update path was unknown. Now, the {microshift-short} version is logged at start up and available with commands such as `journalctl -u microshift | grep "Version"`. (link:https://issues.redhat.com/browse/OCPBUGS-19540[*OCPBUGS-19540*])
```